### DISCUSSION TITLE
deleted start.json :(

### DISCUSSION THREAD

**Discussion by 1001celeritas** [Comfy-Org/ComfyUI#2803] [Q&A]
Hello :), 

I saved START. JSON by accident from the pinokio/comfiUI load page, strangely this deleted the file.

The backup I had was from a previous version of the install. ComfiUI loads but crashes now when I run a prompt.

I'd prefer not to fully reinstall over this one file. What can I do please?

Thank you

-------------------------------->
Here is the backup start.json that doesn't work... the one deleted mentioned xformers and was a bit longer...

{
  "daemon": true,
  "pinokiod": ">=0.1.49",
  "run": [{
    "method": "shell.run",
    "params": {
      "venv": "env",
      "path": "app",
      "env": { "PYTORCH_MPS_HIGH_WATERMARK_RATIO": "0.0" },
      "message": "python main.py {{platform === 'darwin' ? '--force-fp16' : ''}}",
      "on": [{ "event": "/http:\/\/[0-9.:]+/", "done": true }]
    }
  }, {
    "method": "local.set",
    "params": {
      "url": "{{input.event[0]}}"
    }
  }]
}

**Comment by ltdrdata** [comment]
start.json isn't belongs to ComfyUI itself.

The proper place for handling this issue is as follows:
https://github.com/cocktailpeanut/comfyui.pinokio
